recharges/tests.py

```diff
@@ -321,20 +397,21 @@ def test_hotsocket_get_airtime_good(self):
             json.dumps(expected_response_good),
             status=200, content_type='application/json')
 
-        recharge_id = self.make_recharge(msisdn="+277244555", status=0)
+        recharge_id = self.make_recharge(msisdn="+27 711455657", status=0)
         # Execute
         result = hotsocket_get_airtime.delay(recharge_id)
         # Check
-        self.assertEqual(result.get(),
-                         "Recharge for +277244555: Queued at Hotsocket #4487")
+        self.assertEqual(result.get(), "Recharge for +27711455657: Queued at "
+                         "Hotsocket #4487")
         recharge = Recharge.objects.get(id=recharge_id)
-
         self.assertEqual(recharge.status, 1)
         self.assertEqual(recharge.hotsocket_ref, 4487)
         """tests for the correct URL request"""
         self.assertEqual(len(responses.calls), 1)
         self.assertEqual(responses.calls[0].request.url,
                          "http://test-hotsocket/recharge")
+        self.assertEqual(recharge.msisdn, '+27711455657')
+        self.assertEqual(recharge.network_code, 'VOD')
 
```

Comment:
Need a test for a submission that wasn't for a valid network to make sure it doesn't hit hotsocket